migrations/20250624170854_query_statistics_tz.up.sql

```diff
@@ -0,0 +1,29 @@
+-- === UP MIGRATION ===
+
+-- 1. Enable pg_cron extension
+CREATE EXTENSION IF NOT EXISTS pg_cron;
```

Comment:
are you sure this works? i know that we should enable it as well from the AWS RDS configs